test cases/common/223 custom target input extracted objects/check_object.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+import sys, os
+
+if __name__ == '__main__':
+    if len(sys.argv) != 3:
+        print(sys.argv[0], 'object', 'output')
+        sys.exit(1)
+    objExists = os.path.exists(sys.argv[1])
```

Comment:
This should exit immediately if the input file is not found:

```python
if not os.path.exists(sys.argv[1]):
    sys.exit('Input file missing.')
```

Then you don't need to have the `if`s later in the script. Because of the way dependency resolution works, output files should not be written when there are errors.

Reply:
Fair point. I've reworked this now.